src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -435,9 +435,11 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
     ;(async () => {
       const accountAddress = isUtxoChainId(chainId)
         ? await getThorchainSaversPosition({ accountId, assetId })
-            .then(({ asset_address }) =>
-              chainId === bchChainId ? `bitcoincash:${asset_address}` : asset_address,
-            )
+            .then(position => {
+              if (!position) throw new Error(`No position found for assetId: ${assetId}`)
```

Comment:
This looks like overengineered safety for the sake of making the compiler happy, but theoretically, which is absolutely true, but also actually accurate, since you could e.g fully withdraw from THORSwap UI and this would fail